ofl/notosansphagspa/METADATA.pb

```diff
@@ -1,20 +1,29 @@
-name: "Noto Sans Phags Pa"
+name: "Noto Sans PhagsPa"
 designer: "Google"
 license: "OFL"
 category: "SANS_SERIF"
 date_added: "2020-11-19"
 fonts {
-  name: "Noto Sans Phags Pa"
+  name: "Noto Sans PhagsPa"
   style: "normal"
   weight: 400
   filename: "NotoSansPhagsPa-Regular.ttf"
   post_script_name: "NotoSansPhagsPa-Regular"
   full_name: "Noto Sans PhagsPa Regular"
-  copyright: "Copyright 2017 Google Inc. All Rights Reserved."
+  copyright: "Copyright 2022 The Noto Project Authors (https://github.com/notofonts/phags-pa)"
 }
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "math"
 subsets: "menu"
 subsets: "phags-pa"
+subsets: "symbols"
+source {
+  repository_url: "https://github.com/notofonts/phags-pa.git"
+  archive_url: "https://github.com/notofonts/phags-pa/releases/download/NotoSansPhagsPa-v2.004/NotoSansPhagsPa-v2.004.zip"
+}
 is_noto: true
+display_name: "Noto Sans Phags-pa"
```

Comment:
@simoncozens I added the display name. Wikipedia seems to not capitilised "Pa", can you confirm it looks alright to you?

Reply:
Phags-pa is good.